Formula/strongswan.rb

```diff
@@ -1,8 +1,8 @@
 class Strongswan < Formula
   desc "VPN based on IPsec"
   homepage "https://www.strongswan.org"
-  url "https://download.strongswan.org/strongswan-5.9.1.tar.bz2"
-  sha256 "a337c9fb63d973b8440827755c784031648bf423b7114a04918b0b00fd42cafb"
+  url "https://download.strongswan.org/strongswan-5.9.2.tar.bz2"
+  sha256 "61c72f741edb2c1295a7b7ccce0317a104b3f9d39efd04c52cd05b01b55ab063"
```

Comment:
```suggestion
  sha256 "61c72f741edb2c1295a7b7ccce0317a104b3f9d39efd04c52cd05b01b55ab063"
  license "GPL-2.0-or-later"
```
From `src/_copyright/_copyright.c` in the source tarball.